kola/tests/rpmostree/deployments.go

```diff
@@ -199,6 +199,10 @@ func rpmOstreeUpgradeRollback(c cluster.TestCluster) {
 // a package that a) will be around a long time and b) is small +
 // well-defined
 func rpmOstreeInstallUninstall(c cluster.TestCluster) {
+	// bcrypt is available in EPEL and installs on Fedora 29 and RHEL 8
```

Comment:
I just noticed that the comment right above refereces `fpaste` still, so maybe delete that and move this up?

Reply:
ack, will do